test/standalone/addOutArgument/touch-file.zig

```diff
@@ -0,0 +1,15 @@
+const std = @import("std");
+
+pub fn main() !void {
+    var argv_iter = try std.process.ArgIterator.initWithAllocator(std.heap.page_allocator);
+    defer argv_iter.deinit();
+
+    _ = argv_iter.next() orelse @panic("missing arg[0]");
+
+    const argv1 = argv_iter.next() orelse @panic("missing arg[1]");
+
+    var file = try std.fs.createFileAbsolute(argv1, .{});
```

Comment:
This will fail, if the developer wants to run the standalone tests multiple times. Why not remove that file after usage?

Reply:
`createFileAbsolute` will override an existing file and truncate it